datafusion/functions/src/unicode/reverse.rs

```diff
@@ -95,59 +104,58 @@ pub fn reverse<T: OffsetSizeTrait>(args: &[ArrayRef]) -> Result<ArrayRef> {
 
 #[cfg(test)]
 mod tests {
-    use arrow::array::{Array, StringArray};
-    use arrow::datatypes::DataType::Utf8;
+    use arrow::array::{Array, LargeStringArray, StringArray};
+    use arrow::datatypes::DataType::{LargeUtf8, Utf8};
 
     use datafusion_common::{Result, ScalarValue};
     use datafusion_expr::{ColumnarValue, ScalarUDFImpl};
 
     use crate::unicode::reverse::ReverseFunc;
     use crate::utils::test::test_function;
 
+    macro_rules! test_reverse {
```

Comment:
is it possible to have a function rather than macro?

Reply:
Could, sure. I'm curious as to why though. This is just the same approach as I used for lpad.

Reply:
yeah this testing pattern is somewhat common in the string functions

I happen to like .slt in general, but I know that is not for everyone

Reply:
I agree, we trying to use macros if other options didn't work. It takes more time when people debug tests to expand the macros and go through them. 

if its already used in string tests its okay, perhaps we want to revisit macros and replace them with funcs whenever possible

Reply:
Great explanation ,thanks.